samples/matter/lock/README.rst

```diff
@@ -363,6 +363,42 @@ To learn more about factory data, read the :doc:`matter:nrfconnect_factory_data_
 
 .. matter_door_lock_sample_factory_data_end
 
+.. matter_door_lock_sample_factory_data_nrf54h20_start
+
+Merging the factory data hex file with the firmware hex file is currently not available on nRF54H20 DK.
+The factory data support is disabled by default on this board, however you can still use it, but you need to flash the :file:`factory_data.hex` file manually.
+
+To do it complete the following steps in the given order:
+
+1. Flash the :file:`factory_data.hex` file into the device using the following command:
+
+   .. code-block:: console
+
+      nrfutil device program --firmware build/template/zephyr/factory_data.hex
+
+#. Enable factory data support by building the sample with the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA` Kconfig option set to ``y`` using the following command:
+
+   .. code-block:: console
+
+      west build -p -b nrf54h20dk/nrf54h20/cpuapp -- -DCONFIG_CHIP_FACTORY_DATA=y
+
+#. Flash the firmware:
+
+   .. code-block:: console
+
+      west flash --erase
+
+The ``west flash --erase`` command does not clear the factory data partition.
+If you want to clear it, use the following command, and fill the ``<address>`` argument as an address of the factory data partition, by default it is set to ``0xe174000``:
```

Comment:
```suggestion
If you want to clear it, use the following command, and fill the ``<address>`` argument as an address of the factory data partition.
By default, the address is set to ``0xe174000``.
```
Could also be that the argument defaults to the value, not sure.

Reply:
They can modify that value in the dts file, so maybe we should do it generically but point out the default value to ensure that it will work for the first attempt.